app/models/allocation.rb

```diff
@@ -23,6 +23,9 @@ class Allocation < ApplicationRecord
   scope :ongoing, -> {
     where(ongoing: true, user_id: User.active).order(start_at: :desc)
   }
+  scope :hourly_rate_currencies, -> {
+    select(:hourly_rate_currency).distinct.map(&:hourly_rate_currency)
+  }
```

Comment:
This should be a class method since it does not return a relation; in this case, it would be better to use pluck. This way, it does not allocate unnecessary memory.

Reply:
Refactored on f6ff0c5e9dd160c195eaffea20da6b0fc796afd